styles/iconLabel.module.css

```diff
@@ -0,0 +1,22 @@
+.title {
+    width:40%;
+    background:rgba(255, 255, 255, 0.266);
+    padding-left:20px;
+    margin:10px;
+    font-weight:700;
+}
+.value {
+    width:60%;
+    background:rgba(255, 255, 255, 0.266);
+    padding-left:20px;
+    margin:10px;
+}
+.body {
+    display:flex;
+    width:50%;
+}
+.secondaryProfileDesc {
+    margin-left:5px;
+    margin-top:5px;
+    margin-bottom:5px;
+}
```

Comment:
I will wait until we have Material UI incorporated into the project then review these CSS changes (because we will be using `Box` for `div` and `sx` for `className` in Material UI). In another word, we don't need CSS like these anymore, they will all be `sx` (using the css-in-javascript paradigm).